components/dashboard/src/components/UsageView.tsx

```diff
@@ -160,13 +179,61 @@ function UsageView({ attributionId }: UsageViewProps) {
 
     const headerTitle = attributionId.kind === "team" ? "Team Usage" : "Personal Usage";
 
+    const FromLink = forwardRef((arg: any, ref: any) => (
+        <div
+            className="text-blue-500 hover:text-blue-600 dark:text-blue-400 dark:hover:text-blue-500 truncate cursor-pointer"
+            onClick={arg.onClick}
+            ref={ref}
+        >
+            {arg.value}
+        </div>
+    ));
+    const ToLink = forwardRef((arg: any, ref: any) => (
+        <div
+            className="text-blue-500 hover:text-blue-600 dark:text-blue-400 dark:hover:text-blue-500 truncate cursor-pointer"
+            onClick={arg.onClick}
+            ref={ref}
+        >
+            {arg.value}
+        </div>
+    ));
+
     return (
         <>
             <Header
-                title={headerTitle}
-                subtitle={`${new Date(startDateOfBillMonth).toLocaleDateString()} - ${new Date(
-                    endDateOfBillMonth,
-                ).toLocaleDateString()} (updated every 15 minutes).`}
+                title={
+                    <div className="flex items-baseline">
+                        <h1 className="tracking-wide">{headerTitle}</h1>
+                        <h2 className="ml-3">(updated every 15 minutes).</h2>
+                    </div>
+                }
+                subtitle={
+                    <h2 className="tracking-wide flex mt-3">
+                        <h2 className="mr-2">Period: </h2>
```

Comment:
**question**: While the goal long term is to replace the primitive navigation between months with more flexible options like Last 7 days, 30 days, etc with a flexible dropdown (see https://github.com/gitpod-io/gitpod/issues/11685#issuecomment-1197939139) to keep the scope small here, what do you think of moving the period elements below inside the left sidebar? Alternatively, we could keep the custom filtering on the heading subtitle which feels nice.

| Range picker on the sidebar | Range picker on the heading |
|-|-|
| <img width="448" alt="Screenshot 2022-11-09 at 9 22 57 PM" src="https://user-images.githubusercontent.com/120486/200923331-3e5a9faa-63e5-49d3-b624-a275aae94217.png"> | <img width="674" alt="Screenshot 2022-11-09 at 9 27 59 PM" src="https://user-images.githubusercontent.com/120486/200923387-90ef7360-d138-4aaa-8b17-e54d8a9b678c.png"> |

Reply:
If you let me pick, I'd go with the "Selection on the heading" version. Looks slickest to me.
Would be great if you could take a general look at how we want to do the range selection out of the scope of this PR.